lib/jekyll-import/importers/rss.rb

```diff
@@ -52,14 +55,24 @@ def self.write_rss_item(item, options)
         post_name = Jekyll::Utils.slugify(item.title, :mode => "latin")
         name = "#{formatted_date}-#{post_name}"
         audio = render_audio && item.enclosure.url
+        canonical_link = options.fetch("canonical_link", false)
 
         header = {
           "layout" => "post",
-          "title"  => item.title,
-        }
+          "title" => item.title,
+          "canonical_url" => (canonical_link ? item.link : nil),
```

Comment:
Hm, this could create a large site with a lot of `canonical_url: nil` in them since the key is present. It might be best to follow the below "tag" method and do something like:

```ruby
header["canonical_url"] = item.link if canonical_link
```

Reply:
sure, will update it

Reply:
Hi @parkr, we don't have change it to `item.link if canonical_link` as `.compact` will take care of null values